CHANGELOG.md

```diff
@@ -8,6 +8,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added|Changed|Deprecated|Removed|Fixed|Security
 Nothing so far
 
+## 3.0.0 - [unreleased]
+### Added
+- Added support for php 8.
+### Removed
+- Removed support for php 5.
+- Removed everything marked deprecated in the previous major release.
+
```

Comment:
Dit stuk hoeft er nog niet in voor versie 2.x toch?

Reply:
Klopt.  Er staat `unreleased` boven en zit niemand in de weg wat mij betreft